src/TextArea/index.js

```diff
@@ -297,6 +299,7 @@ class Textarea extends React.Component {
     ) : (
       <Wrapper className={'hw-textarea-textarea-wrapper'}>
         <TextArea
+          {...otherProps}
```

Comment:
In conjunction with the comment above, our pattern has been to include this declaration at the end of the prop list.  In most cases, this will be even more of a moot point (particularly once the above comment is addressed), but since it's our pattern, I think we should probably stick to it.  Unless I'm missing something.  Which is likely.